### ISSUE TITLE
Constraint symbols seem to be missing on zoom in

### ISSUE THREAD

**Issue by PLChris** [FreeCAD/FreeCAD#6545]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Forums discussion

https://forum.freecadweb.org/viewtopic.php?f=3&t=66919

### Version

0.20 (Development)

### Full version info

```shell
OS: macOS 10.16
Word size of FreeCAD: 64-bit
Version: 0.20.27936 (Git)
Build type: Release
Python 3.9.10, Qt 5.12.9, Coin 4.0.0, OCC 7.5.3
Locale: C/Default (C)
Installed mods: 
```

### Subproject(s) affected?

Sketcher

### Issue description

On zooming out the constraint symbols can be grouped. On zooming in again this grouping is not undone in all cases. It needs a hover of the mouse over a geometric element or an axis to regroup which then shows all the constraints.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
Hi @abdullahtahiriyo was this issues fixed by #6578 ?

**Comment by abdullahtahiriyo** [comment]
@luzpaz 
I would say it was.
